omod/src/main/webapp/pages/waiverForm.gsp

```diff
@@ -0,0 +1,183 @@
+<%
+    ui.decorateWith("kenyaemr", "standardPage")
+%>
+<html>
+<head>
+    <title>Waive Form</title>
+</head>
+<style>
+body {
+    background: #f3f3f3;
+}
+</style>
+<!--<script type="text/javascript">-->
+<!--	function printReceipt() {-->
+<!--		var printDiv = jQuery("#printDiv").html();-->
+<!--		var printWindow = window.open('height=500,width=400');-->
+<!--		//printWindow.document.write('<html><head><title>Patient Information</title>'); not needed-->
+<!--		printWindow.document.write('<body style="font-family: Dot Matrix Normal,Arial,Helvetica,sans-serif; font-size: 12px; font-style: normal;">');-->
+<!--		printWindow.document.write(printDiv);-->
+<!--		printWindow.document.write('</body>');-->
+<!--		printWindow.document.write('</html>');-->
+<!--		//hide print button-->
+<!--		printWindow.document.getElementById("printSlip").style.visibility = "hidden";-->
+<!--		printWindow.print();-->
+<!--		printWindow.close();-->
+
+<!--	}-->
+
+
+
+
+<!--</script>-->
+<script>
+    function printReceipt (printDiv){
+        var printdata = document.getElementById(printDiv);
+        newwin = window.open("");
+        newwin.document.write(printdata.outerHTML);
+        newwin.print();
+        newwin.close();
+    }
+</script>
+<body>
+<div id="printDiv">
```

Comment:
Each control should have an id and name properties, and again NOT entries will be input fields, other will be read only referencing already existing and captured details from the system. There is a lot already captured that we don't need to repeat here, but only reference them

Reply:
Agin id and name properties should be named uniquely.